### PR TITLE
Build a rpm with 'lein fatrpm'

### PR DESCRIPTION
**Pull request by numbat**
Behaviour should be analogous to the debian package. Based off the fatrpm target from riemann.

### CODE REVIEW

**Comment by pyr** [comment]
Hi @numbat,

Thanks for the quick PR ! Have you been able to install the resulting package ?

**Comment by numbat** [comment]
Hi @pyr, I have worked on CentOS release 6.6 (Final), using it to build and test the installation. It does start the service in the post-install script, this could fail if Cassandra is not started or configured differently to what's expected in the default pithos.yaml file. This is the same behaviour as the debian installer I believe.

**Comment by pyr** [comment]
Hi @numbat,

When running `lein fatrpm` I get the following stacktrace:

```
clojure.lang.Compiler$CompilerException: java.lang.IllegalArgumentException: Malformed member expression, expecting (.member target ...), compiling:(leiningen/fatrpm.clj:182:5)
 at clojure.lang.Compiler.analyzeSeq (Compiler.java:6651)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyze (Compiler.java:6406)
    clojure.lang.Compiler$BodyExpr$Parser.parse (Compiler.java:5782)
    clojure.lang.Compiler$FnMethod.parse (Compiler.java:5217)
    clojure.lang.Compiler$FnExpr.parse (Compiler.java:3846)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6642)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6632)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.access$100 (Compiler.java:38)
    clojure.lang.Compiler$DefExpr$Parser.parse (Compiler.java:538)
    clojure.lang.Compiler.analyzeSeq (Compiler.java:6644)
    clojure.lang.Compiler.analyze (Compiler.java:6445)
    clojure.lang.Compiler.analyze (Compiler.java:6406)
    clojure.lang.Compiler.eval (Compiler.java:6707)
    clojure.lang.Compiler.load (Compiler.java:7130)
    clojure.lang.RT.loadResourceScript (RT.java:370)
    clojure.lang.RT.loadResourceScript (RT.java:361)
    clojure.lang.RT.load (RT.java:440)
    clojure.lang.RT.load (RT.java:411)
    clojure.core$load$fn__5066.invoke (core.clj:5641)
    clojure.core$load.doInvoke (core.clj:5640)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.core$load_one.invoke (core.clj:5446)
    clojure.core$load_lib$fn__5015.invoke (core.clj:5486)
    clojure.core$load_lib.doInvoke (core.clj:5485)
    clojure.lang.RestFn.applyTo (RestFn.java:142)
    clojure.core$apply.invoke (core.clj:626)
    clojure.core$load_libs.doInvoke (core.clj:5524)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invoke (core.clj:626)
    clojure.core$require.doInvoke (core.clj:5607)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    leiningen.core.utils$require_resolve.invoke (utils.clj:68)
    leiningen.core.utils$require_resolve.invoke (utils.clj:71)
    leiningen.core.main$lookup_task_var.invoke (main.clj:61)
    leiningen.core.main$pass_through_help_QMARK_.invoke (main.clj:68)
    leiningen.core.main$task_args.invoke (main.clj:71)
    leiningen.core.main$resolve_and_apply.invoke (main.clj:308)
    leiningen.core.main$_main$fn__6136.invoke (main.clj:377)
    leiningen.core.main$_main.doInvoke (main.clj:366)
    clojure.lang.RestFn.invoke (RestFn.java:408)
    clojure.lang.Var.invoke (Var.java:379)
    clojure.lang.AFn.applyToHelper (AFn.java:154)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.core$apply.invoke (core.clj:624)
    clojure.main$main_opt.invoke (main.clj:315)
    clojure.main$main.doInvoke (main.clj:420)
    clojure.lang.RestFn.invoke (RestFn.java:436)
    clojure.lang.Var.invoke (Var.java:388)
    clojure.lang.AFn.applyToHelper (AFn.java:160)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.main.main (main.java:37)
```

**Comment by numbat** [comment]
Try again - my bad, as I had to transcribe changes between my linux machine and my dev machine.

File: pkg/rpm/init.sh [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# Source function library.
+. /etc/rc.d/init.d/functions
+
+# Pull in sysconfig settings
+[ -f /etc/sysconfig/pithos ] && . /etc/sysconfig/pithos
+
+# PATH should only include /usr/* if it runs after the mountnfs.sh script
+PATH=/sbin:/usr/sbin:/bin:/usr/bin:/usr/local/bin
+DESC="Pithos"
+NAME=pithos
```
Comment (numbat) [inline pkg/rpm/init.sh]: Should this be "pithos-s3" to match the Debian package name?

**Comment by pyr** [comment]
thanks @numbat ! I missed this pending PR.

**Pull request merged**